src/main/java/io/appform/dropwizard/actors/actor/ActorConfig.java

```diff
@@ -79,4 +79,6 @@
     @Valid
     private TtlConfig ttlConfig;
 
+    private long expiryInMs;
```

Comment:
Any reason for scoping this config per actor ? It is better to ask for this per message since this could be dynamic in nature

Reply:
Keep it per message. What is the overhead if ttl is provided per message ?

Reply:
Sure